packages/mui-material-next/src/SnackbarContent/SnackbarContent.tsx

```diff
@@ -52,23 +50,26 @@ const SnackbarContentMessage = styled('div', {
   name: 'MuiSnackbarContent',
   slot: 'Message',
   overridesResolver: (props, styles) => styles.message,
-})({
+})<{ ownerState: SnackbarContentOwnerState }>({
   padding: '8px 0',
 });
 
 const SnackbarContentAction = styled('div', {
   name: 'MuiSnackbarContent',
   slot: 'Action',
   overridesResolver: (props, styles) => styles.action,
-})({
+})<{ ownerState: SnackbarContentOwnerState }>({
   display: 'flex',
   alignItems: 'center',
   marginLeft: 'auto',
-  paddingLeft: 16,
+  paddingLeft: 4,
   marginRight: -8,
 });
 
-const SnackbarContent = React.forwardRef(function SnackbarContent(inProps, ref) {
+const SnackbarContent = React.forwardRef(function SnackbarContent(
```

Comment:
@Best-Sardar Let's implement `useSlotProps` here for the three slots: `root`, `message` and `action`!